src/main/java/com/actionlistapi/model/ActionRequest.java

```diff
@@ -0,0 +1,158 @@
+package com.actionlistapi.model;
+
+import java.util.ArrayList;
+
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+
+import com.actionlistapi.pojo.ActionRequested;
+import com.actionlistapi.pojo.ProcessInstanceStatus;
+import com.actionlistapi.services.ActionRequestedConverter;
+import com.actionlistapi.services.ProcessInstanceStatusConverter;
+import com.actionlistapi.services.RouteNodeConverter;
+
+@Entity
+public class ActionRequest {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY,generator="id_seq")
+	@SequenceGenerator(name="id_seq",sequenceName="id_seq",allocationSize=1)
+	private long id;
+	
+	private String processInstanceId;
+	
+	@ManyToOne
+	private ProcessType processType;
+	
+	private String title;
+	
+	@Convert(converter = ProcessInstanceStatusConverter.class)
+	private ProcessInstanceStatus processInstanceStatus;
+	
+	@Convert(converter = ActionRequestedConverter.class)
+	private ActionRequested actionRequested;
+	
+	private String initiator;
+
+	private String delegator;
+	
+	private String group_;
+	
+	@Convert(converter = RouteNodeConverter.class)
+	private ArrayList<String> routeNodes;
+	
+	private String lastApprovedDate;
+	
+	private String creationDate;
+	
+
+	public String getProcessInstanceId() {
```

Comment:
You might consider bringing in [Lombok](https://projectlombok.org/) so you don't have to write so much boilerplate.